server/services/transaction.service.js

```diff
@@ -146,3 +149,21 @@ export const getTotalAmountPaidByUser = async (userId) =>
       },
     },
   ]);
+
+export const getReferralRewards = async (userId) =>
```

Comment:
I think this should be for contribution rewards 😏 

```suggestion
export const getContributionRewards = async (userId) =>
```